cds/utils.bal

```diff
@@ -0,0 +1,192 @@
+import ballerina/http;
```

Comment:
Missing licence. Add in other places as well

Reply:
Added